.github/scripts/trymerge.py

```diff
@@ -981,6 +983,28 @@ def merge_into(self, repo: GitRepo, *,
             for pr in additional_merged_prs:
                 pr.add_numbered_label("merged")
 
+        if comment_id and self.pr_num:
+            # Finally, upload the record to Rockset. The list of pending and failed
+            # checks are at the time of the merge
+            save_merge_record(
+                collection=ROCKSET_MERGES_COLLECTION,
+                comment_id=comment_id,
+                pr_num=self.pr_num,
+                owner=self.org,
+                project=self.project,
+                pending_checks=pending_checks,
+                failed_checks=failed_checks,
+                is_failed=False,
+                dry_run=dry_run,
+                skip_mandatory_checks=skip_mandatory_checks,
+                ignore_current=bool(ignore_current_checks),
+                error="",
```

Comment:
I think you can just not set the error field and send it to rockset and itll put null in